bin/vagrant_provision.sh

```diff
@@ -1,71 +1,113 @@
 #!/bin/bash
 
 set -e
+set -x
 
 INSTALL_DIR=/home/vagrant
 
 cd $INSTALL_DIR/kitsune
 
+# Configure locales
+export LC_ALL="en_US.UTF-8"
+locale-gen en_US.UTF-8
+
 # Install package for add-apt-repository
 apt-get install -y software-properties-common
 
-# ElasticSearch key add/setup
-curl http://packages.elasticsearch.org/GPG-KEY-elasticsearch | apt-key add -
-echo "deb http://packages.elasticsearch.org/elasticsearch/0.90/debian stable main" \
-        > /etc/apt/sources.list.d/elasticsearch.list
-
 # Add the needed repositories for Node/Redis/Python2.6
 add-apt-repository ppa:chris-lea/node.js
 add-apt-repository ppa:chris-lea/redis-server
 
 apt-get update
 
-# Set default password for MariaDB
-export DEBIAN_FRONTEND=noninteractive
-debconf-set-selections <<< 'mariadb-server-5.5 mysql-server/root_password password rootpass'
-debconf-set-selections <<< 'mariadb-server-5.5 mysql-server/root_password_again password rootpass'
+# Install Python development-related things
+apt-get install -y -q libapache2-mod-wsgi python-pip python-virtualenv libpython-dev
 
 # Install services and their dependencies
-# Services: Sphinx, MariaDB, ElasticSearch, Redis, and Memcached
-apt-get install -y sphinx-common libapache2-mod-wsgi python-pip libmysqlclient-dev git \
-                   libxml2-dev libxslt1-dev zlib1g-dev libjpeg-dev python-dev libssl-dev \
-                   openjdk-7-jre-headless mariadb-server-5.5 nodejs elasticsearch redis-server \
-                   memcached libffi-dev
-
-# Setup the virtualenv and start using it
-pip install virtualenv
-virtualenv $INSTALL_DIR/virtualenv
-chown -R vagrant $INSTALL_DIR/virtualenv
-source $INSTALL_DIR/virtualenv/bin/activate
-
-pip install -U 'pip<7'
-$INSTALL_DIR/kitsune/peep.sh install -r $INSTALL_DIR/kitsune/requirements/default.txt
-$INSTALL_DIR/kitsune/peep.sh install -r $INSTALL_DIR/kitsune/requirements/dev.txt
+# Services: Sphinx, Redis, and Memcached
+apt-get install -y libxml2-dev libxslt1-dev zlib1g-dev libjpeg-dev libssl-dev \
+                   nodejs redis-server memcached libffi-dev
 
-# Copy configurations for kitsune and mysql
-# MySQL Default User: root
-# MySQL Default Pass: rootpass
-cp $INSTALL_DIR/kitsune/configs/vagrant/settings_local.py \
-   $INSTALL_DIR/kitsune/kitsune/settings_local.py
+# Install git
+apt-get install -y -q git
 
-cp $INSTALL_DIR/kitsune/configs/vagrant/my.cnf /etc/mysql/my.cnf
+# Install mysql
```

Comment:
I think kitsune uses ``mariadb server``. so we should install ``mariadb`` instead of ``mysql``